openslides/utils/models.py

```diff
@@ -54,3 +54,36 @@ def get_rest_pk(self):
         the database pk.
         """
         return self.pk
+
+    def save(self, skip_autoupdate=False, information=None, *args, **kwargs):
+        """
+        Calls the django save-method and afterwards hits the autoupdate system.
+
+        If skip_autoupdate is set to True, then the autoupdate system is not
+        informed about the model changed. This also means, that the model cache
+        is not updated.
+
+        The optional argument information can be an object that is given to the
+        autoupdate system. It should be a dict.
+        """
+        # TODO: Fix circular imports
+        from .autoupdate import inform_changed_data
+        return_value = super().save(*args, **kwargs)
+        inform_changed_data(self, information=information)
+        return return_value
+
+    def delete(self, skip_autoupdate=False, information=None, *args, **kwargs):
+        """
+        Calls the django delete-method and afterwards hits the autoupdate system.
+
+        See the save method above.
+        """
+        # TODO: Fix circular imports
+        from .autoupdate import inform_changed_data
+        # Django sets the pk of the instance to None after deleting it. But
+        # we need the pk to tell the autoupdate system which element was deleted.
+        instance_pk = self.pk
```

Comment:
Are you sure that we (or Django) do not expect anywhere that pk is None if a model instance is deleted?